deploy/hub/clusterrole.yaml

```diff
@@ -18,6 +18,10 @@ rules:
 - apiGroups: ["rbac.authorization.k8s.io"]
   resources: ["clusterroles", "clusterrolebindings", "roles", "rolebindings"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+# Allow hub to manage coordination.k8s.io/lease
```

Comment:
the name of this file should change to better match the name of the resource

Reply:
renamed to `hub-controller-clusterrole.yaml`